server/controllers/request.js

```diff
@@ -0,0 +1,81 @@
+const Request = require("../models/Request");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /request
+// @desc create a new pet sitting request
+// @access Private
+exports.makeRequest = asyncHandler(async (req, res) => {
+  const body = req.body;
+  if (!(body || body.sitterId || body.ownerId || body.duration)) {
```

Comment:
Optional: You can resort to validating the request body and even path parameters using middleware - this way we can apply this middleware to the applicable routes 😄